lib/channels/src/index.ts

```diff
@@ -73,10 +70,10 @@ export class Channel {
     }
 
     const handler = () => {
+      this.handleEvent(event);
```

Comment:
What's the rationale?

Reply:
Hmm it's been a while.. I did have a reason for this. it was something timing related, or catching an error.

I'll check what happens if I reverse it again.

Reply:
nothing happens, I'll revert this.